### PR TITLE
HttpHeaderModifier 

### PR DESCRIPTION
**Pull request by pjainxido**
dev 환경에서만 사용하는 컴포넌트
http request header 옵션 커스텀 가능하게

### CODE REVIEW

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by lcc3108** [comment]
deploy success [url](http://moomark-frontend-pr-21.micalgenus.com/)

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by lcc3108** [comment]
deploy success [url](http://moomark-frontend-pr-21.micalgenus.com/)

File: src/components/GlobalHeader/HttpHeaderModifier/HttpHeaderModifier.module.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    margin: 1rem;
+    border-radius: 4px;
+    padding: 0.5rem;
+    color: var(--text-primary);
+    background-color: var(--accent);
+  }
+  .resetButton {
+    background-color: var(--accent);
+  }
+  .result{
```
Comment (lcc3108) [inline src/components/GlobalHeader/HttpHeaderModifier/HttpHeaderModifier.module.scss]:
```suggestion
  .result {
```

**Review by lcc3108 — APPROVED** [review]

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
deploy success [url](http://moomark-frontend-pr-21.micalgenus.com)

**Comment by blackcowmooo** [comment]
Update repo fail

**Pull request merged**